website/content/contributors/_index.md

```diff
@@ -10,51 +10,60 @@ Welcome! Are you interested in contributing to one of CNCF hosted projects? This
 
-The Cloud Native Computing Foundation supports the worldwide community of the Cloud Native meetups. They are listed on [meetup.com](https://www.meetup.com/pro/cncf/) website.
+Although TAGs are designed to focus on specific areas, you can begin contributing through working groups (WGs) that operate under the TAGs with even more narrow scopes. WGs are community-driven groups that encourage ongoing collaboration in specific areas of interest.
 
-CNCF is currently working on expanding the Cloud Native community around the globe, and we are happy to accept the new meetup communities to join our network, and become one of the official CNCF meetups.
+More details about the CNCF TAGs is available [here](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md). You can find the list of TAGs [here](https://github.com/cncf/toc/tree/main/tags).
 
-Details on the Meetups program, together with the best practices on running CNCF Meetups is available [here](https://github.com/cncf/meetups).
 
-### Mentorship Programs
+### Community Groups
 
-The Cloud Native Computing Foundation participates in various mentoring programs, including:
+The Cloud Native Computing Foundation supports the worldwide community of the Cloud Native Community Groups (CNCGs). They are listed on [community.cncf.io](https://community.cncf.io/) website. CNCF is currently working on expanding the Cloud Native community worldwide, and we are happy to accept any new local communities to join our network.
 
-- [LFX Mentorship](https://github.com/cncf/mentoring/tree/master/lfx-mentorship) (previously known as Community Bridge) by the Linux Foundation;
-- [Google Summer of Code](https://github.com/cncf/mentoring/tree/master/summerofcode) (GSoC);
-- [Google Season of Docs](https://github.com/cncf/mentoring/tree/master/seasonofdocs) (GSoD);
-- [Outreachy](https://github.com/cncf/mentoring/tree/master/outreachy)
+Are you passionate about Cloud Native technologies and interested in starting a community group in your area? Look no further than the CNCF Meetups program! Check out the [CNCF Community Groups repository](https://github.com/cncf/communitygroups) for more information about the program, including best practices for running successful CNCF Community Groups.
+
+### Ambassadors
+
+[Cloud Native Ambassadors](https://www.cncf.io/people/ambassadors/) (CNAs) are individuals who are passionate about [Cloud Native Computing Foundation](https://www.cncf.io/) technology and projects, recognized for their expertise, and willing to help others learn about the framework and community.
+
+Successful ambassadors are people such as bloggers, influencers, evangelists who are already engaged with a CNCF project in some way including contributing to forums, online groups, community events, etc.
+
+Details on the Ambassadors program, and information on how to join CNCF as an Ambassador is available [here](https://github.com/cncf/ambassadors).
```

Comment:
Same comment as above, it would be more accessible to not use "here" in the link text.

Reply:
Sounds good. I've replaced it